price-estimator/src/main.rs

```diff
@@ -98,6 +103,47 @@ struct Options {
     /// an estimate and the solver submitting a solution.
     #[structopt(long, env = "EXTRA_ROUNDING_BUFFER_FACTOR", default_value = "2.0")]
     extra_rounding_buffer_factor: f64,
+
+    // These are copies of the same arguments used in the driver for economic viability.
+    // It would be nice if we could avoid copy pasting these. Maybe structopt/clap has a feature
+    // like serde's flatten that would allow us to do this.
+    #[structopt(
+        long,
+        env = "ECONOMIC_VIABILITY_SUBSIDY_FACTOR",
+        default_value = "10.0"
+    )]
+    economic_viability_subsidy_factor: f64,
+    #[structopt(
+        long,
+        env = "ECONOMIC_VIABILITY_MIN_AVG_FEE_FACTOR",
+        default_value = "1.1"
+    )]
+    economic_viability_min_avg_fee_factor: f64,
+    #[structopt(long, env = "MIN_AVG_FEE_PER_ORDER", default_value = "0")]
+    fallback_min_avg_fee_per_order: u128,
```

Comment:
Do you think these parameters should have doc comments as well so they show up in the `--help`?

Reply:
In this case I intentionally removed them as a way of saying "the documentation for this is given in the driver" so that I would copy paste less. Hopefully the doc comments can be part of the deduplication effort. I don't mind adding the doc comments back either but this was my initial decision.

Reply:
If its going to be addressed in the de-duplication effort, then we don't really need to add it here to.